x-pack/plugin/ilm/src/main/java/org/elasticsearch/xpack/ilm/action/TransportPutLifecycleAction.java

```diff
@@ -121,21 +125,44 @@ public static class UpdateLifecyclePolicyTask extends AckedClusterStateUpdateTas
         private final Map<String, String> filteredHeaders;
         private final NamedXContentRegistry xContentRegistry;
         private final Client client;
+        private final boolean verboseLogging;
 
         public UpdateLifecyclePolicyTask(
             Request request,
             ActionListener<AcknowledgedResponse> listener,
             XPackLicenseState licenseState,
             Map<String, String> filteredHeaders,
             NamedXContentRegistry xContentRegistry,
-            Client client
+            Client client,
+            boolean verboseLogging
```

Comment:
Maybe remove the `verboseLogging` param here? 
and just set the `verboseLogging` to `true`?
Then the other constructor can't reuse this constructor, but
then for regular usage of this class, verbose logging is guaranteed to be enabled.